test/Microsoft.Azure.Devices.Edge.Test.Common/IEdgeDaemon.cs

```diff
@@ -15,7 +15,7 @@ public enum EdgeDaemonStatus
 
     public interface IEdgeDaemon
     {
-        Task InstallAsync(string deviceConnectionString, Option<string> packagesPath, Option<Uri> proxy, CancellationToken token);
+        Task InstallAsync(Option<string> packagesPath, Option<Uri> proxy, CancellationToken token);
```

Comment:
Actually we would catch them because of `set -e`. Admittedly the semantics of `set -e` are complicated (for pipelines, compound lists, AND-or lists, etc), so appending `|| exit $?` might be more straightforward.

The looked through all our multi-line PowerShell commands, and they're ok. Once you remove commands that set system environment variables and commands that will throw terminating errors (like the dot-sourcing operator), your only left with a single command.

Reply:
Done

Reply:
Thank you.

Reply:
Danke